.github/workflows/ruby.yml

```diff
@@ -67,3 +64,6 @@ jobs:
 
       - name: StandardRB
         run: bundle exec standardrb --format simple
+
+      - name: JavaScript Audit
```

Comment:
I'm guessing the `Javascript Audit` is replacing the `Install yarn dependencies`?

Reply:
Not quite. `Install yarn dependencies` was basically just loading all the JavaScript packages into the `node_modules` directory. With Rails 7, we don't have to do that anymore. The `config/importmap.rb` file specifies all the JS files/packages we need and where to find them. This cuts out a LOT of JS dependencies compared to the previous approach.

`JavaScript Audit` here is just making sure that the versions of our JS packages don't have any security issues, which we've never done in this repository before.

Reply:
See https://github.com/rails/importmap-rails for more information on how the new way is set up.